frontend/src/components/navbar.tsx

```diff
@@ -133,6 +161,14 @@ class NavigationBar extends Component<NavigationBarProp, NavigationBarState> {
               <img src={tornjak_logo} height="50" width="160" alt="Tornjak" /></a>
           </span>
         </div>
+        {/* Temporarily using trust domain as server unique identifier */}
+        <div className="spire-server-unique-identifier">
+          <Tag type="cyan">
+            <span style={{ fontWeight: 'bold' }}>Server ID: </span>
+            {this.props.globalServerInfo.trustDomain}
+            <span style={{ textDecoration: 'underline', fontWeight: 'bold' }}>{this.props.globalDebugServerInfo.svid_chain[0].id.path}</span>
```

Comment:
Nit: Because it seems most spire-server paths will be `/spire/server` I would maybe bold the trust domain instead - i think it's likely at the end the more important/distinguishing piece of information

Reply:
The design was made assuming the server name might be unique, especially when servers are in the same trust domain. If it is the case that the spire-server paths will be` /spire/server`, I agree. I have made the change.